.github/dependabot.yml

```
@@ -1,11 +1,15 @@
 version: 2
 
 updates:
+  # npm should be used for npm and yarn
   - package-ecosystem: npm
     open-pull-requests-limit: 10
     directory: "/"
     schedule:
       interval: "weekly"
       day: "sunday"
```

Can we sneak this in? I think Monday morning save us a few thoughts about the website on the weekend.

```suggestion
    schedule:
      interval: 'weekly'
      day: "monday"
      time: '06:00'
      timezone: 'Europe/Berlin'
```